### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by kim-fitness**
Add some mandatory pre-steps before building the docker image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +29,10 @@ export KUBECONFIG=</path/to/kubeconfig>
 
 Build the docker image to run the placement controller.
 ```sh
+mkdir -p $GOPATH/src/github.com/open-cluster-management-io
+cd $GOPATH/src/github.com/open-cluster-management-io
+git clone https://github.com/open-cluster-management-io/placement.git
+cd placement
```
Comment (elgnay) [inline README.md:35]: @kim-fitness I tested in my environment. The image can be built without those steps. Why do you think they are mandatory?

By the way, in this readme, we assume that user has already cloned this repo to local disk, and the current working dir is the directory of the repo.

Comment (elgnay) [reply README.md:35]: Anyway, I think it's fine to add one step before `Deploy the placement controller`, like

### Clone this repo
```sh
git clone https://github.com/open-cluster-management-io/placement.git
cd placement
```

### Deploy the placement controller
... ...

Comment (kim-fitness) [reply README.md:35]: Hi @elgnay ,
Thanks for the feedback. It is OK to remove the following two steps from the README file and make the image building success. Let me make a new commit to this PR. 
```
mkdir -p $GOPATH/src/github.com/open-cluster-management-io
cd $GOPATH/src/github.com/open-cluster-management-io
```

**Comment by xunpan** [comment]
@kim-fitness 
please avoid adding your local binaries in the PR.

**Comment by qiujian16** [comment]
Thanks, please squash the commit and signoff.

**Comment by qiujian16** [comment]
btw, I would suggest to add `_output` into .gitignore.

**Comment by kim-fitness** [comment]
> @kim-fitness
> please avoid adding your local binaries in the PR.

Oops, that's my fault, let me reset this commit then, thanks for the heads-up!

**Comment by kim-fitness** [comment]
Since the modification of file `.gitignore` is not related with the original issue, I'll leave it as a separate commit.
If it is not the practice of this project, please let me know and squash them up then. Thanks.

**Comment by qiujian16** [comment]
that is fine, would you squash this to one commit and we are good to go.

**Pull request merged**